cmd/cephfs/main.go

```diff
@@ -42,8 +35,27 @@ var (
 )
 
 func main() {
+	if err := flag.Set("logtostderr", "true"); err != nil {
```

Comment:
make a util function for this?

Reply:
will do

Reply:
I'm getting
```
E0212 15:15:23.815436    5850 log.go:38] failed to set log_backtrace_at flag: syntax error: expect file.go:234
```
if I put it in the `ceph-csi/pkg/util` package - I'm not 100% sure why...i guess because it's in another package?

Reply:
Nevermind, should be working now...I had to ignore error checking when copying args from glog to klog though.